instruqt/closing-information-gap/04-adding-to-the-chart/check-shell

```diff
@@ -6,14 +6,14 @@ set -euxo pipefail
 result=0
 
 # look for the preflight template file
-if [[ ! -f /home/replicant/harbor/templates/troubleshoot/support-bundle.yaml ]] ; then
-  fail-message 'Please create the support bundle spec template file in the Harbor Helm chart template directory'
+if [[ ! -f /home/replicant/slackernews/templates/troubleshoot/support-bundle.yaml ]] ; then
+  fail-message 'Please create the support bundle spec template file in the Slackernews Helm chart template directory'
   let "result = result + 1"
 fi
 
-if [[ "$(helm template /home/replicant/harbor | yq 'select( .kind == "Secret" ) | select( .metadata.labels."troubleshoot.sh/kind" == "support-bundle" ) | .stringData | has("support-bundle-spec")' | head -1)" == "true" ]] ; then
+if [[ "$(helm template /home/replicant/slackernews | yq 'select( .kind == "Secret" ) | select( .metadata.labels."troubleshoot.sh/kind" == "support-bundle" ) | .stringData | has("support-bundle-spec")' | head -1)" == "true" ]] ; then
   set +e pipefail
-  analyzer_results="$(helm template /home/replicant/harbor | kubectl support-bundle --interactive=false --kubeconfig /home/replicant/.kube/config -)"
+  analyzer_results="$(helm template /home/replicant/slackernews | kubectl support-bundle --interactive=false --kubeconfig /home/replicant/.kube/config -)"
   set -e pipefail
 else
   fail-message 'Please add your support bundle definition to the support bundle template file'
```

Comment:
I'm hitting this fail-message but I _think_ i have the assignment completed correctly

<img width="1734" alt="Screenshot 2024-04-25 at 17 06 08" src="https://github.com/replicatedhq/replicated-field-labs/assets/292598/e088004a-b4e3-4cb4-a5a1-61bf56c13215">

Reply:
I've seen a few of these in the past. Sometimes it's my check being too tight, some times it's the instuctions being unclear. I'll look into it.

Reply:
I'm running a bunch of tests around the solve/check loop for this one and I _think_ my debugging there will clear this issue.

Reply:
I think I have this one sorted now. It was also checking whether you did the install incorrectly, but you didn't get the far because of the first failure.